great_expectations/datasource/data_connector/configured_asset_azure_data_connector.py

```diff
@@ -0,0 +1,146 @@
+import logging
+import os
+from typing import List, Optional
+
+import great_expectations.exceptions as ge_exceptions
+
+try:
+    from azure.storage.blob import BlobServiceClient
+except ImportError:
+    azure.storage.blob = None  # type: ignore [name-defined]
```

Comment:
Ensuring basic `mypy` coverage for this file.

Reply:
@cdkini Just a quick thought -- in this try block, should we not nullify the entire "azure" object?  Or are we already certain that `azure.storage` is not NULL?  Thanks!

Reply:
That's a great point! I'll nullify the entire `azure` object